Enabling Third-party Smart Contract Account Applications on Polkadot Based on Account Abstraction.md

```diff
@@ -0,0 +1,159 @@
+
+### Team Code Repos
+
+https://github.com/aawallet/aa-wallet-web-app
+
+
+Please also provide the GitHub accounts of all team members. If they contain no activity, references to projects hosted elsewhere or live are also fine.
+
+- https://github.com/mxnares/market
+- http://github.com/zhangzhishun
+
+
+### Team LinkedIn Profiles (if available)
+
+- https://www.linkedin.com/in/myronzhang1024/
+- https://www.linkedin.com/in/erika-wang-ba5844241/
+- https://www.linkedin.com/in/miguelmonares/
+
+## Development Status :open_book:
+
+We have finished the initial research and design, and we are developing the first version of our project. We are also currently looking for grants, support, advice, and partnerships.
+
+- Demo: https://smarter.web3-idea.xyz
+- Prototype: https://www.figma.com/file/4Yu3MRAu1EG0kQMInFmnJk/Smart-AA-Wallet-Prototype?type=design&node-id=0%3A1&mode=design&t=WJmYKNbNmoL5HPNA-1
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+- **Total Estimated Duration:** 6 months
+- **Full-Time Equivalent (FTE):**  4 FTE
+- **Total Costs:** $28,000
+
+### Milestone 1 — AA Standard Enabling Third-party Account Applications
+
+- **Estimated duration:** 2 months
+- **FTE:**  3
+- **Costs:** 10,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | Apache 2.0 |
+| **0b.** | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a developer can develop his/her third-party account applications via our standard. |
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests to verify our AA standard can fully support third-party account applications. |
+| **0d.** | Docker | We will provide Dockerfiles that can be used to test all the functionality delivered with this milestone. |
+| 0e. | Article | We will publish an article that explains how our AA standard works and runs for developers to smoothly provide users with different account applications in their AA accounts. |
+| 1. | User interface | We will design and develop a sample AA wallet for users to have their AA account and equip it with third-party account applications. |
+| 2. | Smart contracts | We will deliver a set of ink! smart contracts to show the specifications of our AA standard supporting third-party account applications. |
+| 3. | Substrate module: transaction execution | We will create a Substrate module to define and achieve the interfaces of executing transactions. |
+| 4. | Substrate module: identity verification | We will create a Substrate module to define and achieve the interfaces of verifying the identity of users. |
```

Comment:
What's the purpose of identifying the identity of users? What identity? What do you mean by "achieve the interfaces of executing transactions"? How do the smart contracts and pallets interact in this architecture?